### PR TITLE
add admin view to reprocess errors forms

### PR DESCRIPTION
**Pull request by snopoke**
http://manage.dimagi.com/default.asp?250459

FYI @amstone326

### CODE REVIEW

**Comment by dimagimon** [comment]
There are 6 errors:

* corehq/apps/hqadmin/views.py, line 13 - F401 'FormProcessorCouch' imported but unused
* corehq/apps/hqadmin/views.py, line 14 - F401 'FormProcessorInterface' imported but unused
* corehq/apps/hqadmin/views.py, line 15 - F401 'SubmissionPost' imported but unused
* corehq/apps/hqadmin/views.py, line 16 - F401 'should_use_sql_backend' imported but unused
* corehq/apps/hqadmin/views.py, line 51 - F401 'FormAccessorSQL' imported but unused
F401 'LedgerAccessorSQL' imported but unused
* corehq/apps/hqadmin/views.py, line 52 - F401 'FormAccessorCouch' imported but unused

**Comment by dimagimon** [comment]
There are 9 errors:

* corehq/apps/cleanup/management/commands/reprocess_error_forms.py, line 5 - F401 'CommandError' imported but unused
* corehq/apps/cleanup/management/commands/reprocess_error_forms.py, line 13 - F401 'string_to_datetime' imported but unused
* corehq/apps/cleanup/management/commands/reprocess_error_forms.py, line 74 - F821 undefined name 'since'
* corehq/apps/hqadmin/views.py, line 13 - F401 'FormProcessorCouch' imported but unused
* corehq/apps/hqadmin/views.py, line 14 - F401 'FormProcessorInterface' imported but unused
* corehq/apps/hqadmin/views.py, line 15 - F401 'SubmissionPost' imported but unused
* corehq/apps/hqadmin/views.py, line 16 - F401 'should_use_sql_backend' imported but unused
* corehq/apps/hqadmin/views.py, line 51 - F401 'FormAccessorSQL' imported but unused
F401 'LedgerAccessorSQL' imported but unused
* corehq/apps/hqadmin/views.py, line 52 - F401 'FormAccessorCouch' imported but unused

File: corehq/apps/cleanup/management/commands/reprocess_error_forms.py [hunk 1]
```diff
@@ -85,32 +34,41 @@ def add_arguments(self, parser):
...
+                  (len(succeeded), len(succeeded) + len(failed), len(failed)))
+            if error_messages:
+                print("The following errors were seen: \n%s" %
+                      ("\n".join("%s: %s" % (v, k) for k, v in error_messages.items())))
+
+    def _get_form_ids(self, domain):
+        if should_use_sql_backend(domain):
+            problem_ids = FormAccessorSQL.get_form_ids_in_domain_by_type(domain, 'XFormError')
+        else:
+            problem_ids = get_form_ids_by_type(domain, 'XFormError', start=since)
```
Comment (dimagimon) [inline corehq/apps/cleanup/management/commands/reprocess_error_forms.py]: F821 undefined name 'since'

**Comment by amstone326** [comment]
@snopoke Nice!! Will this be a mode within the Raw Forms, Errors & Duplicates report, or another view entirely?

**Review by mkangia — APPROVED** [review]
Don't have much context on this but otherwise changes look fine.

**Comment by snopoke** [comment]
updated to allow re-processing from the submission error report (under the `support` toggle)

**Pull request merged**